src/main/kotlin/com/ampnet/web3provider/service/impl/AccountInformationServiceImpl.kt

```diff
@@ -19,20 +19,20 @@ class AccountInformationServiceImpl(
     companion object : KLogging()
 
     override fun getBalance(request: JsonRpcRequest): ProviderResponse {
-        if (request.params.isEmpty()) return defaultProviderService.getResponse(request)
-        val address = request.params[0]
+        val address = request.params.getOrNull(0)
         val blockParameter = request.params.getOrNull(1) ?: DefaultBlockParameterName.LATEST.value
         logger().info { "Received request to get ${request.method} for address: $address and block: $blockParameter" }
```

Comment:
```suggestion
        logger.info { "Received request to get ${request.method} for address: $address and block: $blockParameter" }
```

Reply:
Try to only one version, either `logger.` or `logger().`. I would prefer the first one.